### DISCUSSION TITLE
Agent Descriptions (version 0.2.2 onwards) and team setup.

### DISCUSSION THREAD

**Discussion by AssetDK** [microsoft/autogen#1109] [Ideas]
As described in this blogpost [All About Agent Descriptions](https://microsoft.github.io/autogen/blog/2023/12/29/AgentDescriptions), UserProxyAgent and AssistantAgent takes a new description parameter. This seems to work really nice. 

However I will use this post to add some ideas for how you can make the team setup more specific to your needs...

**Comment by rickyloynd-microsoft** [comment]
@afourney
